1.10/deploying-services/gpu.md

```diff
@@ -0,0 +1,169 @@
+---
+post_title: Enabling GPU Support
+feature_maturity: preview
+menu_order: 110
+---
+
+DC/OS supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads. 
+
+# Enabling GPUs
+GPUs must be enabled during DC/OS installation. Follow the instructions to enable GPUs based on your DC/OS deployment method.
+
+## Enabling GPUs for a Custom DC/OS Installation
+
+1.  Install DC/OS using the [custom installation instructions](/docs/1.10/installing/custom/advanced/) with the `enable-gpu-isolation: 'true'` configuration parameter specified.
+1.  Install the [NVIDIA Management Library (NVML)](https://developer.nvidia.com/nvidia-management-library-nvml) on each node of your cluster that has GPUs, unless it is already installed. Find detailed installation instructions [here](https://github.com/apache/mesos/blob/master/docs/gpu-support.md#external-dependencies).
+
+## Enabling GPUs for an AWS EC2 DC/OS Installation
+
+###  Prerequisites
+- The AWS DC/OS advanced template [system requirements](/docs/1.10/installing/cloud/aws/advanced/).
+- The `zen.sh` script copied to your local machine. The script and instructions are [here](/docs/1.10/installing/cloud/aws/advanced/).
+
+### Create Dependencies
+
+1. Run the `zen.sh` script to create the Zen template dependencies. These dependencies will be used as input to create your stack in CloudFormation.
+  ```
+  bash ./zen.sh <stack-name>
+  ```
+  **Important:** You must run the `zen.sh` script before performing the next steps.
+
+1. Follow the instructions [here](/docs/1.10/installing/cloud/aws/advanced/) to create a cluster with advanced AWS templates, using the following GPU-specific configuration.
+
+1. On the **Create Stack** > **Specify Details** page, specify your stack information and click **Next**. Here are the GPU-specific settings.
+  - **CustomAMI** - Specify the custom AMI for your region:
+
+      - us-west-2: `ami-9b5d97fb`
+      - us-east-1: `ami-e10e50f6`
+      - ap-southeast-2: `ami-37b28f54`
+  - **MasterInstanceType** - Accept the default master instance type (e.g. `m3.xlarge`).
+  - **PrivateAgentInstanceType** - Specify a machine type of `g2.2xlarge`. This is the GPU-supported machine type.
```

Comment:
These don't necessarily have to be `g2.2xlarge` instances. They can be any type of Nvidia GPU that AWS supports. This includes:
```
Model	GPUs	vCPU	Mem (GiB)	SSD Storage (GB)
g2.2xlarge	1	8	15	1 x 60
g2.8xlarge	4	32	60	2 x 120
p2.xlarge1	1	4	61	12
p2.8xlarge	8	32	488	96
p2.16xlarge	16	64	732	192
```